contrib/prometheus/README

```diff
@@ -5,15 +5,20 @@ To use it, first install prometheus by following the instructions at
 
   http://prometheus.io/
 
-Then add a new job to the main prometheus.conf file:
+Then add a new job to the main prometheus.yml file:
```

Comment:
prometheus.yaml, no?

Reply:
`prometheus.yml` is working in my running setup

Reply:
`prometheus.yaml` is working in my running setup